docs/cli_reference.md

```diff
@@ -4074,6 +4074,8 @@ The VM or VMs on which this operation is performed are selected using the standa
 
 Use the `force` argument to cause an ungraceful shutdown, similar to pulling the plug on a physical server.
 
+An HVM mode VM requires `force=true` to be shutdown, unless [Guest Tools](guests.md#guest-tools) have been installed.
```

Comment:
It is my understanding that installing the guest tools is a MUST in any case, so I would rather phrase it in a way that emphases this, rather than make it sound the default is there are no tools and you need force=true.

Reply:
Maybe simply "HVM mode VMs require the guest tools to be installed for a graceful shutdown to be possible."

Reply:
Out of curiosity, what makes it a must?  I've actually never installed guest tools on any of the VMs I develop against.

I'm happy to do the reword, just curious.

Reply:
* it makes the VM able to communicate its IP address to the XAPI
* it's necessary for clean shutdown or reboot from the host
* can be necessary for live migration
* on Windows VMs, they include drivers without which you will not get good performance (except in rare cases)